_data/collection-index.yml

```
@@ -923,3 +923,8 @@
   contact: https://github.com/lx-0/devcontainer-templates/issues
   repository: https://github.com/lx-0/devcontainer-templates
   ociReference: ghcr.io/lx-0/devcontainer-templates
+- name: AIT-Assistive-Autonomous-Systems Dev Container Features
+  maintainer: AIT-Assistive-Autonomous-Systems maintainers
+  contact: https://github.com/AIT-Assistive-Autonomous-Systems/devcontainer_features/issues
+  repository: https://github.com/AIT-Assistive-Autonomous-Systems/devcontainer_features
+  ociReference: ghcr.io/AIT-Assistive-Autonomous-Systems/devcontainer_features
```

```suggestion
  ociReference: ghcr.io/ait-assistive-autonomous-systems/devcontainer_features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

---

Hi,
updated.